### PR TITLE
Change module path to classic-terra/core

### PR DESCRIPTION
**Pull request by inon-man**
## Summary of changes
Original PR is https://github.com/classic-terra/core/pull/43, Note: protobuf changes were not included.

This change is needed for me to do "go get github.com/classic-terra/classic" in my db-benchmark. If kept "github.com/terra-money/core", go would fetch Terra 2.0

## Report of required housekeeping

- [ ] Github issue OR spec proposal link
- [ ] Wrote tests
- [ ] Updated API documentation (client/lcd/swagger-ui/swagger.yaml)
- [ ] Added a relevant changelog entry: clog add [section] [stanza] [message]

----

## (FOR ADMIN) Before merging

- [ ] Added appropriate labels to PR
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" (coding standards)
- [ ] Confirm added tests are consistent with the intended behavior of changes
- [ ] Ensure all tests pass

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -7,7 +7,7 @@ LEDGER_ENABLED ?= true
 BINDIR ?= $(GOPATH)/bin
 BUILDDIR ?= $(CURDIR)/build
 SIMAPP = ./app
-HTTPS_GIT := https://github.com/terra-money/core.git
+HTTPS_GIT := https://github.com/terra-money/classic-core.git
```
Comment (nghuyenthevinh2000) [inline Makefile:10]: I thought this will be: "classic-terra/core" ?

Comment (StrathCole) [reply Makefile:10]: It's the TFL repo.

Comment (inon-man) [reply Makefile:10]: It is intended since our repo has no valid main

**Comment by inon-man** [comment]
Oh sorry that I accidentally pushed merge button. If you guys find any issue, I will make a revert PR

**Pull request merged**